### DISCUSSION TITLE
Do you want a Flashlight app?

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#248] [Polls]
If yes, please comment your reasons below.

**Comment by NGRhodes** [comment]
Yes, but its not essential.

**Comment by FerrusFox** [comment]
I use it for the bright screen function frequently.

**Comment by antodc** [comment]
Very low priority, in my view.

**Comment by DKB-91** [comment]
Yes, but it is not in any way high priority.

**Comment by olifre** [comment]
Yes, I did not find a good and simple alternative for flashlight via bright screen, and it's very lightweight and easy to use. 
It's very helpful in cases when you do not want to use the very bright, physical flashlight, but the screen flashlight, which is usually darker / can be tuned and warmer.

**Comment by poeticus** [comment]
Yes, particularly for the bright screen functionality. Setting this to red then turning the screen off means that I can have a low impact light at night with a single press of my phone's on/off button. 
Not only do I find this preferable when waking at night to the harsh light of the standard torch, but it is invaluable for any activity, such as star gazing or astrophotography, where it is important that once eyes have taken time to adapt to the dark, this night vision is not then disrupted.

**Comment by naveensingh** [comment]
I was skeptical at first and yes, it's not a priority but I have decided to release Flashlight.
